app/Http/Controllers/PagesController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Blog;
+use App\Blogcomment;
+class PagesController extends Controller
+{
+    public function home()
+     {
+         $show = Blog::all();
+         $show = Blog::paginate(4);
+        return view('welcome',compact('show'));
```

Comment:
indentation

Also, keep one line break in logical code blocks. (return is separate code block). This make reading code easier.